torch/_inductor/fx_passes/quantization.py

```diff
@@ -704,13 +705,7 @@ def qconv_binary(match: Match, *args, **kwargs):
 
 
 def _register_quantization_unary_fusion():
-    from .mkldnn_fusion import (
-        _gelu_fusion_1 as _gelu_fusion_erf,
-        _gelu_fusion_2 as _gelu_fusion_tanh,
-        _hardswish_fusion,
-        _hardtanh_fusion,
-        _silu_fusion,
-    )
+    from .mkldnn_fusion import _hardswish_fusion, _hardtanh_fusion, _silu_fusion
```

Comment:
why do we still need to register these fusion patterns in the post grad fusion passes? Per the PR discussion, should it all be moved to the freezing passes?